source/common/config/watch_map.cc

```diff
@@ -242,6 +242,11 @@ void WatchMap::onConfigUpdate(
       cur_watch->callbacks_.onConfigUpdate({}, {}, system_version_info);
     }
   }
+
+  // Processing point when resources are successfully ingested.
+  if (xds_config_tracker_.has_value()) {
+    xds_config_tracker_->onConfigIngested(type_url_, decoded_resources, removed_resources);
+  }
```

Comment:
Why not do this inside the new_grpc_mux_impl?

Reply:
Changed. For `legacy` and `unified` mux, I think the subscription state is the best place to put the `onConfigAccepted`. It is called right after the watch map callbacks, which represents the actual accepted resources.
Before I am thinking to pass the decoded_resources, but for delta we can put under `handleGoodResponse`.

Reply:
PTAL. Thank you!